recipes-soc/rdk-gstreamer-utils-platform/rdk-gstreamer-utils-platform.bb

```diff
@@ -0,0 +1,29 @@
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=86d3f3a95c324c9479bd8986968f4327"
+
+DEPENDS += "${@bb.utils.contains('DISTRO_FEATURES', 'gstreamer1', 'gstreamer1.0  gstreamer1.0-plugins-base', 'gstreamer gst-plugins-base', d)}"
+DEPENDS += "rdk-gstreamer-utils-headers"
+
+SRC_URI = "git://git@github.com/ssadha725_comcast/rdk-gstreamer-utils-raspberrypi;${RDKE_GITHUB_SRC_URI_SUFFIX}"
```

Comment:
Change the repo to https://github.com/rdkcentral/rdk-gstreamer-utils-raspberrypi/

Reply:
Done